src/basemaps.js

```diff
@@ -1,26 +1,26 @@
 /**
- * Type of carto.basemaps.voyager
+ * Voyager vector basemap
  *
- * @typedef {string} CARTO Voyager style
  * @memberof carto.basemaps
+ * @name voyager
  * @api
  */
 export const voyager = 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json';
 
 /**
- * Type of carto.basemaps.darkmatter
+ * Dark-Matter vector basemap
  *
- * @typedef {string} CARTO Dark Matter style
  * @memberof carto.basemaps
+ * @name darkmatter
  * @api
  */
 export const darkmatter = 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json';
 
 /**
- * Type of carto.basemaps.positron
+ * Positron vector basemap
  *
- * @typedef {string} CARTO Positron style
  * @memberof carto.basemaps
+ * @name positron
```

Comment:
Isn't the name extracted automatically?

Reply:
We are using it to generate the documentation.